### PR TITLE
Added bitonic subarray #7322

### PR DESCRIPTION
**Pull request by ishitakapoor26**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

### Description
Added Bitonic Subarray Code. A bitonic subarray is a subarray in which elements are first increasing and then decreasing. User enters array elements/numbers and program returns length of largest bitonic subarray.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#7322

### CODE REVIEW

**Comment by ishitakapoor26** [comment]
Pls review my pr @SKAUL05 @nrickles3 @ArchanGhosh @Ayushjain2205

File: C/cp/Bitonic_subarray.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    // print the longest bitonic subarray
+    printf("The length of the longest bitonic subarray is %d\n", lbs_len);
+    printf("The longest bitonic subarray is [%d, %d]", beg, end);
+ 
+    return lbs_len;
+}
+ 
+int main(void)
+{
+    int A[] = { 3, 5, 8, 4, 5, 9, 10, 8, 5, 3, 4 };
```
Comment (siddharth25pandey) [inline C/cp/Bitonic_subarray.c]: Use Dynamic Input

File: C/cp/Bitonic_subarray.c [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    int n = sizeof(A) / sizeof(A[0]);
+ 
+    findBitonicSubarray(A, n - 1);
```
Comment (siddharth25pandey) [inline C/cp/Bitonic_subarray.c]: Display a message in print form as what are you showing output to the user

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
Do the suggested changes and also add the update in README file

**Comment by ishitakapoor26** [comment]
@siddharth25pandey pls review changes.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format 
- Remove extra lines of code and spaces
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add a new line at the EOF
- Use proper display message how you are reading and writing outputs
- Add input-output prompts to make the program more interactive.
- Make sure the comments you add are above the line of code and not alongside it.

**Comment by ishitakapoor26** [comment]
> Few changes:
> 
> * Add Multi-Line Comments
> * Add a beginner-friendly description at the top
> * Use proper code Format
> * Remove extra lines of code and spaces
> * Add sample I/O at the end of the Code
> * Add Time and Space Complexity at the end of the Code
> * Add a new line at the EOF
> * Use proper display message how you are reading and writing outputs
> * Add input-output prompts to make the program more interactive.
> * Make sure the comments you add are above the line of code and not alongside it
I've already made these changes. Pls review my pr @siddharth25pandey

**Review by siddharth25pandey — APPROVED** [review]
@ishitakapoor26 LGTM 💯 Able to merge

**Pull request merged**